packages/lib/modules/pool/actions/create/steps/details/useReClammConfigurationOptions.tsx

```diff
@@ -191,5 +191,10 @@ export function useReClammConfigurationOptions(): ReClammConfigOptionsGroup[] {
     }
   }, [isInitialReClammConfig])
 
+  useEffect(() => {
+    if (!initialTargetPrice || !priceRangePercentage) return
+    updatePriceBounds(initialTargetPrice, priceRangePercentage)
+  }, [initialTargetPrice, priceRangePercentage])
+
```

Comment:
why is this necessary when `updatePriceBounds` is already triggered by the `targetPrice.updateFn` and  and `priceRangeBoundaries.updateFn` ?

Reply:
because people don't want a pure radio component i have reverted my changes again
let's just use your implementation for now